### ISSUE TITLE
Missing inputs - transaction fails

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2272]
This one is while on lbryumx1 after claiming a few rewards and trying to send.  I have the blockchain/headers file if needed. A couple of us have run into this recently.  Clearing out db file / resync usually helps. 

```2019-06-27 16:28:46,160 INFO     lbrynet.wallet.resolve:163: fetching certificate to check claim signature
2019-06-27 16:28:46,311 WARNING  torba.client.basenetwork:30: Wallet server returned an error. Code: 1 Message: the transaction was rejected by network rules.

Missing inputs
[0100000006a451c3b7dc1287a84e3f6c0a763cb90df70df83d151863ae48b7ee3d502b9de3010000006a4730440220533c31397773714cda71b91d40ff69b1b55169f4827c205b5d8d7ffa564d017f022022f4b27e0b2fa136666ca7ac955e6556d72b0dd8cc215a2b0f3408de8f00a24f012103e811a23de9e915cd616dbd1a327eaf1a79f4c3dd19457a22de0080f7d268e66dffffffffc26524c3c7ab5113622c830abbf5e3157191233c6092acc73dda7902e4ea3fb0010000006b483045022100f217cf3177cc1c67bf4cc8e7d20f159a9e1fcbc45f49bbc5c8017b7f6cbc09c4022067af14b9a4482cc06d90387be76a2a81208bc8c6d66357c3afc3781951fed97e01210223122ee22abb1cda1010fdf29476d6298ac1909b1cbc8afc3bd5bde351d20ffdffffffff4e8efbd969b86c6d914288d219cecc7886c9fb89c858df4764e074dcae997493010000006b483045022100f8af1200f96e15efbd7bc9c6cfc620edb696556c37e8780cce4815aff5c41c620220539d6cbe3e3b54c61246f3becca3297b901a7ebefdd567825bb7706cbd88cb3701210273c05974f1ec5f4d37bb2bd8d31a5ff78aab916de1a91ee2742aa2aec30d6f4effffffff6567b809a2d01acb3f744155c29cb954f5b76eb016caace9c19dc27d6adf1893010000006b483045022100db23626fcb86dfa568b58828cb4eb685b2a095b9fd692192596755f197ba3240022026ca064f954dc8addf35aafe8e57a524f5bccab2a3fe94cee9249104da80325c0121036590c9e72e0555d0c06f5eca9c4a62e4a977177facceb5af495f3501ed300485ffffffff31dbb1828d21690608d6ddfe15a65ffed2ec0074e590018805d0d07354ce88f3000000006b483045022100ff51f2d06a0a33a383f725a6f8e4016529464cbec6036f0ff9ab844c9d1cc3c00220336335dc9b1f3500ed0ae62e6f565bdf3b9c809d8826dbbb30c752aca745ee850121021d86a7a1055e44dd4c05ed3b3bd00aa6106ff6451c949b5841e915bcf55e78e6ffffffffb2e0754e0525d271409dbf1dfbe232a462f8f22d3c54338de4c86a17921c56ba010000006b483045022100f4e170e08b2153ea68b4da8ae39a3aaf4ded69f700a4f3017161b16f8093d4d50220667fe8ae23eaceab6262106ea6ceb74cbc891f1dddba65d02512621a03ab1e5c01210210ca2e2702c7a6232f6733b1fd038f298c5391b384bc5582a0bbb7a5f9c945baffffffff0200f90295000000005cb62a417354686553756e526973657354686553756e416c736f5365747353686565744d75736963566964656f146bc359288614e9b3360dd635eb02252c69406aad6d7576a914fc2be3f60b70526e3dc46c651541028cf60ab6b088ac05729e11000000001976a914696d766ca2e48cccf74ffa0bc9727439315bb8f588ac00000000]
2019-06-27 16:28:46,328 ERROR    lbrynet.extras.daemon.Daemon:576: error handling api request
Traceback (most recent call last):
  File "lbrynet\extras\daemon\Daemon.py", line 570, in _process_rpc_call
  File "lbrynet\extras\daemon\Daemon.py", line 2687, in jsonrpc_support_create
  File "lbrynet\extras\daemon\Daemon.py", line 3421, in broadcast_or_release
  File "site-packages\torba\client\basenetwork.py", line 31, in send_request
  File "site-packages\torba\client\basenetwork.py", line 28, in send_request
  File "site-packages\torba\rpc\session.py", line 481, in send_request
torba.rpc.jsonrpc.RPCError: (1, 'the transaction was rejected by network rules.\n\nMissing inputs\n[0100000006a451c3b7dc1287a84e3f6c0a763cb90df70df83d151863ae48b7ee3d502b9de3010000006a4730440220533c31397773714cda71b91d40ff69b1b55169f4827c205b5d8d7ffa564d017f022022f4b27e0b2fa136666ca7ac955e6556d72b0dd8cc215a2b0f3408de8f00a24f012103e811a23de9e915cd616dbd1a327eaf1a79f4c3dd19457a22de0080f7d268e66dffffffffc26524c3c7ab5113622c830abbf5e3157191233c6092acc73dda7902e4ea3fb0010000006b483045022100f217cf3177cc1c67bf4cc8e7d20f159a9e1fcbc45f49bbc5c8017b7f6cbc09c4022067af14b9a4482cc06d90387be76a2a81208bc8c6d66357c3afc3781951fed97e01210223122ee22abb1cda1010fdf29476d6298ac1909b1cbc8afc3bd5bde351d20ffdffffffff4e8efbd969b86c6d914288d219cecc7886c9fb89c858df4764e074dcae997493010000006b483045022100f8af1200f96e15efbd7bc9c6cfc620edb696556c37e8780cce4815aff5c41c620220539d6cbe3e3b54c61246f3becca3297b901a7ebefdd567825bb7706cbd88cb3701210273c05974f1ec5f4d37bb2bd8d31a5ff78aab916de1a91ee2742aa2aec30d6f4effffffff6567b809a2d01acb3f744155c29cb954f5b76eb016caace9c19dc27d6adf1893010000006b483045022100db23626fcb86dfa568b58828cb4eb685b2a095b9fd692192596755f197ba3240022026ca064f954dc8addf35aafe8e57a524f5bccab2a3fe94cee9249104da80325c0121036590c9e72e0555d0c06f5eca9c4a62e4a977177facceb5af495f3501ed300485ffffffff31dbb1828d21690608d6ddfe15a65ffed2ec0074e590018805d0d07354ce88f3000000006b483045022100ff51f2d06a0a33a383f725a6f8e4016529464cbec6036f0ff9ab844c9d1cc3c00220336335dc9b1f3500ed0ae62e6f565bdf3b9c809d8826dbbb30c752aca745ee850121021d86a7a1055e44dd4c05ed3b3bd00aa6106ff6451c949b5841e915bcf55e78e6ffffffffb2e0754e0525d271409dbf1dfbe232a462f8f22d3c54338de4c86a17921c56ba010000006b483045022100f4e170e08b2153ea68b4da8ae39a3aaf4ded69f700a4f3017161b16f8093d4d50220667fe8ae23eaceab6262106ea6ceb74cbc891f1dddba65d02512621a03ab1e5c01210210ca2e2702c7a6232f6733b1fd038f298c5391b384bc5582a0bbb7a5f9c945baffffffff0200f90295000000005cb62a417354686553756e526973657354686553756e416c736f5365747353686565744d75736963566964656f146bc359288614e9b3360dd635eb02252c69406aad6d7576a914fc2be3f60b70526e3dc46c651541028cf60ab6b088ac05729e11000000001976a914696d766ca2e48cccf74ffa0bc9727439315bb8f588ac00000000]')
2019-06-27 16:28:50,047 WARNING  lbrynet.blob_exchange.client:125: f65382ebaaa65deaf79a1c07759e8131a6af1352bd17b9708a575b58e5e98297171b62d1bc426e5e777b870d748cbf8d not in availability response from 24.184.108.248:3335
2019-06-27 16:28:50,047 WARNING  lbrynet.blob_exchange.client:126: {'blob_data_payment_rate': 'RATE_ACCEPTED', 'available_blobs': [], 'lbrycrd_address': 'bTUkxDm7PPy8BU794Zjya2Nbt8A4M2VzWZ'}
2019-06-27 16:28:52,596 INFO```

**Comment by tzarebczan** [comment]
This is still happening in 0.38.5: 
```User(651248) Desktop Error: "(1, 'the transaction was rejected by network rules.\\n\\nMissing inputs\\n[01000000023df00f8fe07279ccc4ac5ed4f65bbd8122e196436ebb1af5be898fc6c11d3808000000006a47304402204ff28896dac9dc653290f427a0b0b89aad648bb274dd5c6969c69c87f56425bf02203c6e8ef43453c5f5b5b91ef474ae4483e3454708b8ebd94be66ff0813a2910b2012103637be2d7f89683e7f56f82b8d017c676fdab0aaee6221d5dccbb2fb2dc67b548ffffffff048fb48e037b498171bc23441552cbc18547118e2b5297428c773ece59a07f55000000006a4730440220247013f474e1c14ac1f23f72561986cdefb4a47d2eb724fa0fa53473d0e5c78b0220296f38a09e17de736eca756e128aa3b960f1b9a5aa9a9bfd576b0f9df8820c22012102f9666ced030e93a5956f523d3a48621069fcc8e36130420c769dac891bc94c45ffffffff028096980000000000fd9e01b70c5175696574566963746f727914bde2e109f73d1a0b7fdc0541811fdf7d3e262c4d4d5d01000ade010a9c010a30c47d6d85a9e24f7f70a23c111e7cff8b6f74082b4d9e7abda513832922c65321fee0294fe54c6a27e34f0b2fb32dee441221517569657420566963746f7279206368616e67657320283229202831292e70646618af8d06220f6170706c69636174696f6e2f706466323061f2c83671d2ea04f3f6aab0f430415d3396888e2d6c05149cd7c531a2717054374e5df0b7abc19edf9b2650bf78a08a1a13416c6c2072696768747320726573657276656428808defe80532220801121955155089785ca79d2c282a1fe89b7a8aba8ca55229072f39091880c2d72f420d517569657420566963746f72794a345468697320697320612068656172747761726d696e672073686f72742073746f727920666f7220796f756e67206164756c74732e52302a2e68747470733a2f2f737065652e63682f632f487a425747486c304a63647364743046696e4d35626231582e6a7067620208016d6d76a914155089785ca79d2c282a1fe89b7a8aba8ca5522988acfed29700000000001976a9146843653a06236103e295214657c40e09c629807b88ac00000000]')"```

**Comment by tzarebczan** [comment]
Another: ```Desktop Error: "(1, 'the transaction was rejected by network rules.\\n\\nMissing inputs\\n[0100000001eb8645f0297a386f35608902b5acd8adebaa973998ac35ee9c1380fa05d0cb67010000006b483045022100be2e0ac59eb2686ee69b6c3c54fdff9f299d3597214a79a097975034d775fc0902205d9c7ddeca1c2a6c03b470ccd07d495f3f301bfd8936219b76723998c26b47ff01210259fdd604cce891b6071e662f6c5b476a2a1aa517d6f40e99958c5105f3e568e3ffffffff0240420f0000000000fdeb01b50b4c4252596d6174757265334dc001013839f1f357bc0ec1ff3898f9102ee063b1abbe47c2e96093c905bafbc90bd2dae6a5e953260bb215f142a3342c7c8051648676c98241bb8743adb3a6f32bfc24fa8bc05da5df90504a48b97d0de684e7965112570a9f010a85010a30caf732f103b781f3c3170deabba4ccd7f70c86557b2c25075cd20226b7ebee9c12952c2a93693fca9d0a4c485c7a64af120f4c4252596d6174757265332e6d703418a2dcbc102209766964656f2f6d703432308dfe80369a136985f83e4fcff22f59352a6beba442e6513a2b5bb3c04447e4cc89fc82e49de456714f744554ddc96ad21a044e6f6e6528b1c487ea055a0908800510e80218f804422e57484f204953205348453f3f20414d415a494e4720424f44592047555953207c7c204d41545552452c20504f524e4a0f544950204c425259204d415455524552302a2e68747470733a2f2f737065652e63682f382f33704e416350456b4d3756554176536c5956696e4a6376622e6769665a066d61747572655a044e5346575a04504f524e5a044749524c5a086c6561726e696e675a09456475636174696f6e5a0566756e6e795a0d456e7465727461696e6d656e745a0b706f702063756c74757265620208016d7576a914434aadd0e14e6067c9aed1f7e28240b1d1726e6b88acc032a634000000001976a9148d04bd617e8951fa4de9a9ee965c0ae992fdd2b388ac00000000]')"```

**Comment by tzarebczan** [comment]
Another: ```User(xxx) Desktop Error: "(1, 'the transaction was rejected by network rules.\\n\\nMissing inputs\\n[01000000016641012a69736ff541d8f71b0664303b8c24b8769ecf04d8b12a1063fc0442e0010000006a47304402201509a01fe0787be8fce53e487db0c87460597c6582a4e247866a9fcb746c1834022068b10d55b9b28ac852721ca1844633b07c509a45a1d8d8730b9516fb2e9a8645012102148e4801c0d75cee395646f73e7f99bd018efa66de90d01340979eafb8b0d498ffffffff028096980000000000fd8102b5274465657053746174652d497341637469766174696e675468656972536c656570657243656c6c734d3a02011638dae336a857f97d64c20a933e3e5425b1e3cd144173cd12af9fb42ccf44878a0e0ab747f47fc07ac90e71eb330e23fa9bf49ce27d3afc67e9b3d92d594c3154f8e4ab112abc0d53abfb853cca22d782ac5fad0ac0010aa6010a3038f5f9949a15ef76784de0a23b6872fb715fc8e75f4ca8a6ff91f32847afabd3e56e4833fe35e64ae7746ca3713881d71230446565702053746174652049732041637469766174696e6720546865697220536c65657065722043656c6c732e6d703418d2f189652209766964656f2f6d70343230cb13619e740e64c96cc96883ec2bad8a43695c36b2b35265f91778a4c567b987d20cbb75dcd3711e659f8ba95484a59c1a044e6f6e6528d8d7a2ea055a0908800a10d005189b06422c446565702053746174652049732041637469766174696e6720546865697220536c65657065722043656c6c734abb01446565702053746174652049732041637469766174696e6720546865697220536c65657065722043656c6c730a0a4c656f205a6167616d69206578706f736573207468652067616d65706c616e206f662074686520656c69746520746f2073746f6b65206120636976696c2077617220696e20416d65726963612e0a0a68747470733a2f2f7777772e696e666f776172732e636f6d2f77617463682f3f766964656f3d35643438613865383632393265363030313238323234333652302a2e68747470733a2f2f737065652e63682f652f3163714a37416637754b7a72466b417174344b304c55646b2e6a7067620208016d7576a914fb25d4a6cacb3538a4e27c31d3446d190b85775e88ac9ca2ab01000000001976a9147b98a33715443e3c17754bc082318dab00cbdd5f88ac00000000]')"```

**Comment by lyoshenka** [comment]
jack thinks this is a sync issue. might be fixed by #2744. but he didn't sound confident.
